recipe/build.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/bash
 
+set -ex
+
 cat > site.cfg <<EOF
 [DEFAULT]
 libraries = blas,cblas,lapack
 library_dirs = $PREFIX/lib
 include_dirs = $PREFIX/include
 EOF
 
+# CFLAGS=-ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O3 -pipe -I$PREFIX/include -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=${PREFIX}=/usr/local/src/conda-prefix
+CFLAGS="$CFLAGS -O2"
+CPPFLAGS="$CPPFLAGS -O2"
```

Comment:
Can you do this only for aarch64? And also try going back to azure.

Reply:
mostly done. gotta see what MACHINE is for aarch64